server/src/services/teacher.services.js

```diff
@@ -26,14 +26,18 @@ export const getAllTeachers = () => {
 export const getTeacherById = (teacher_id) => {
 
     return new Promise((resolve, reject) => {
-        const sql = "SELECT T.id as id, T.cod_group as cod_group, G.title_group as title_group, G.cod_department as cod_department FROM Teachers T, Groups G WHERE T.id = ? AND T.cod_group = G.cod_group";
+        const sql = "SELECT T.id as id, T.name as name, T.surname as surname, T.email as email, T.cod_group as cod_group, G.title_group as title_group, G.cod_department as cod_department " +
```

Comment:
In this file the green part can be accepted because are the updates that I asked you yesterday